src/schema/v2/user.ts

```diff
@@ -52,7 +52,15 @@ export const UserAdminNotesField: GraphQLFieldConfig<any, ResolverContext> = {
       )
     }
 
-    return await userAdminNotesLoader(id)
+    const adminNotes = await userAdminNotesLoader(id)
+
+    const sortedAdminNotes = adminNotes.sort((objA, objB) => {
```

Comment:
#minor #notblocking could be nice for Gravity to offer sorting of these, or offer them w/ a default sort of rev. chron. order

Reply:
Yeah. Good call! Will focus on finishing out the remaining MP / Forque side of things, and may come back to this.